utils/src/main/kotlin/FileMatcher.kt

```diff
@@ -31,6 +31,11 @@ class FileMatcher(
      * [1]: https://docs.oracle.com/javase/tutorial/essential/io/fileOps.html#glob
      */
     val patterns: List<String>,
+
+    /**
+     * Toggle the case-sensitivity of the matching.
+     */
+    caseSensitive: Boolean = true
```

Comment:
E.g. Kotlins `equals()` uses `ignoreCase` which defaults to `false`. I think I'd like it better to align on that.

Reply:
Done in a commit on top.